### PR TITLE
8248-CRUD progress

### PR DESCRIPTION
**Pull request by bapboy18**

### CODE REVIEW

File: app/controllers/admin/progresses_controller.rb [hunk 1]
```diff
@@ -0,0 +1,45 @@
+class Admin::ProgressesController < ApplicationController
+  load_and_authorize_resource
+
+  def index
```
Comment (thiennv57) [inline app/controllers/admin/progresses_controller.rb]: not neccessary

**Comment by vuducluan** [comment]
OK

**Comment by NgocVu1994** [comment]
OK

**Comment by NgocVu1994** [comment]
OK

**Pull request merged**